mobile_app_connector/models/app_hub.py

```diff
@@ -49,6 +49,13 @@ def mobile_get_message(self, partner_id, **pagination):
         sponsorships = partner.sponsorship_ids.filtered('is_active')
         children = sponsorships.mapped('child_id')
 
+        letters = self.env['correspondence'].search([
+            ('direction', '=', 'Beneficiary To Supporter'),
```

Comment:
Here you limit to child letters. Are there no tile for displaying the sponsor letters as well?

Reply:
for now, I did not need another letter